Bottom: d73730d5264f1238ad703e2b47c5fa4c238049a4
Top:    1870d653567fe4cdcf60f7ad563387ea8c8d23ef
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-12 14:10:06 +0000

Not sure what this patch is for ;)


---

diff --git a/src/draw.c b/src/draw.c
index 11c75d2..98ab484 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -530,7 +530,7 @@ DrawEverything (BoxTypePtr drawn_area)
 	r_search (PCB->Data->element_tree, drawn_area, NULL, EMark_callback,
 		  NULL);
       /* Draw rat lines on top */
-      if (PCB->RatOn)
+      if (PCB->RatOn && gui->set_layer ("rats", SL (RATS, 0), 0))
 	DrawRats(drawn_area);
     }
